### PR TITLE
Cleans up ruins

### PR DESCRIPTION
**Pull request by Latentish**
## About The Pull Request
We as maptainers had a meeting on cleaning up the ruins in the game. As is our most spawned planets had the lowest quality ruins overall. Shiptest has inherited a lot of ruins that simply do not meet up to our newest standards, and are taking up the only ruin spawn on each planet. While multi-ruin spawning is in the pipeworks, we as maptainers have unanimously agreed to remove these maps.
This  PR also adds a new folder with maps that are being deprecated for either not working properly, or being too unbalanced to keep in the game. We want to see these maps reworked before being introduced back into the game. In this folder is a non-compiled .dm file that houses their datums for spawning, effectively removing them from spawning.
This also removes all content relating to the kinggoat. (not including achievements)

## Why It's Good For The Game
This PR removes a lot ruins that are either broken, wholly unbalanced, unfun, or poorly designed/mapped. This not only cuts down on these maps significantly, it also more evenly distributes the ratios of ruins per planet, which is inextricably tied to which planets spawn the most frequently, and how often. This has been a long time coming, but now it's been done carefully with the total approval by all 3 maptainers, and various admins and code maintainers.

## Changelog
:cl:phoaly
add: Adds folder of deprecated ruins and datums.
del: Removes various low quality ruins, attached areas, and datums.
/:cl:

### CODE REVIEW

**Review by Fikou — CHANGES_REQUESTED** [review]
Delete the king goat ruin and all the code content (the king goat).

**Comment by triplezeta** [comment]
> Delete the king goat ruin and all the code content (the king goat).

TRUE

**Comment by tmtmtl30** [comment]
death to the goat

**Comment by Zevotech** [comment]
Jungle Witch was personally redone by me, I'd suggest giving its new layout a look and telling me what you think before removing it.

**Comment by Latentish** [comment]
> Jungle Witch was personally redone by me, I'd suggest giving its new layout a look and telling me what you think before removing it.

A few reasons actually.
We came to the conclusion that while it isn't a horrible ruin, there's just not much to it aside from an enemy, some minor loot, and hazards in a square. I wouldn't be against having it set aside as a sub-ruin once multi-ruin generation happens, but it would need some work.
There's also this fun addition that made this decision a bit easier? I don't know if this is intended, or if it's a really bad coincidence. But this seems to communicate something something not good. A person, their cat, a sentience potion, and a love potion.
image

File: code/datums/achievements/boss_achievements.dm [hunk 1]
```diff
@@ -66,12 +66,6 @@
 	desc = "You've now ruined years of mythical storytelling."
 	database_id = BOSS_MEDAL_WENDIGO
 
-/datum/award/achievement/boss/king_goat_kill
-	name = "King Goat Killer"
-	desc = "The king is dead, long live the king!"
-	database_id = BOSS_MEDAL_KINGGOAT
-	icon = "goatboss"
```
Comment (Fikou) [inline code/datums/achievements/boss_achievements.dm]: there is no support of removing achievements iirc so bring these back

File: code/modules/mob/living/simple_animal/hostile/megafauna/king_of_goats.dm [hunk 2]
```diff
@@ -1,343 +0,0 @@
...
-
-		if(3)
-			adjustBruteLoss(25)
-
-/mob/living/simple_animal/hostile/megafauna/king/phase2
-	name = "emperor of the goats"
-	desc = "The King of Kings, God amongst men, and your superior in every way."
-	icon_state = "king_goat2"
-	icon_living = "king_goat2"
-	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 4, /obj/item/clothing/head/goatpelt/king = 1)
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/hostile/megafauna/king_of_goats.dm:76]: did you remove this goatpelt thing

**Comment by Zevotech** [comment]
> > Jungle Witch was personally redone by me, I'd suggest giving its new layout a look and telling me what you think before removing it.
> 
> A few reasons actually. We came to the conclusion that while it isn't a horrible ruin, there's just not much to it aside from an enemy, some minor loot, and hazards in a square. I wouldn't be against having it set aside as a sub-ruin once multi-ruin generation happens, but it would need some work. There's also this fun addition that made this decision a bit easier? I don't know if this is intended, or if it's a really bad coincidence. But this seems to communicate something something not good. A person, their cat, a sentience potion, and a love potion.

This Is Not The Remapped Version. This IS:
image
Way larger, has a decent amount of challenge and a reasonable reward.

File: code/modules/mob/living/simple_animal/hostile/megafauna/king_of_goats.dm (hunk 2 above)
Comment (Latentish) [reply code/modules/mob/living/simple_animal/hostile/megafauna/king_of_goats.dm:76]: yup

File: code/datums/achievements/boss_achievements.dm (hunk 1 above)
Comment (Latentish) [reply code/datums/achievements/boss_achievements.dm]: fixed

**Comment by Latentish** [comment]
> This Is Not The Remapped Version. This IS: image Way larger, has a decent amount of challenge and a reasonable reward.

I've changed this.

**Review by MarkSuckerberg — APPROVED** [review]
Seems good to me :+1:

**Review by Fikou — APPROVED** [review]

**Pull request merged**